pkg/testframework/console-operator.go

```diff
@@ -210,3 +210,51 @@ func SetLogLevel(t *testing.T, client *Clientset, logLevel operatorsv1.LogLevel)
 func GenerationChanged(oldGeneration, newGeneration int64) bool {
 	return oldGeneration == newGeneration
 }
+
+// the operator is settled if conditions match:
+// - available: true
+// - progressing: false
+// - degraded: false
+func operatorIsSettled(operatorConfig *operatorsv1.Console) bool {
+	settled := true
+	for _, condition := range operatorConfig.Status.Conditions {
+		if condition.Type == operatorsv1.OperatorStatusTypeAvailable {
+			if condition.Status == operatorsv1.ConditionFalse {
+				settled = false
```

Comment:
I think we should `break` if we know that the there is even a single condition that is not making the operator settled, right ?

```suggestion
				settled = false
				break
```

Reply:
These are very finite, like, maximum 5.  I didn't think it was worth the extra lines of code :)

Reply:
updated! cant hurt.